views/incorrect-information/wrong-statement-of-capital.html

```diff
@@ -26,14 +26,12 @@ <h1 class="govuk-heading-xl">
             <p>The company's share capital does not match the shares held by its shareholders.</p>
           {% endif %}
 
-          <p>You must update this information by filing a confirmation statement through
-            <a target="_blank" rel="noopener noreferrer" href="https://ewf.companieshouse.gov.uk//seclogin?tc=1" data-event-id="webfiling-service-link-from-invalid-share-or-amount-unpaid-totals-wrong-statement-of-capital-page">our WebFiling service</a>. You will need to sign in to or create a WebFiling account to do this.</p>
-
-        {% else %}
-          <p>Currently, changes to share capital can only be made by filing a confirmation statement through
-            <a target="_blank" rel="noopener noreferrer" href="https://ewf.companieshouse.gov.uk//seclogin?tc=1" data-event-id="webfiling-service-link-from-wrong-statement-of-capital-page">our WebFiling service (opens in a new tab)</a>. You will need to sign in to or create a WebFiling account to do this.
-          </p>
         {% endif %}
+
+        <p>You must update this information by filing a confirmation statement through
+          <a href="https://ewf.companieshouse.gov.uk//seclogin?tc=1">our WebFiling service</a>. You will need to sign in to or create a WebFiling account to do this.
```

Comment:
the old links used to open in a new tab, so we should do the same with this new url unless the story tells us not to.